### DISCUSSION TITLE
VITE_PROXY 配置已经无效了？如果不能使用建议从文档删除

### DISCUSSION THREAD

**Discussion by GreatAuk** [vbenjs/vue-vben-admin#2914] [Q&A]
全局搜索也没看到 `VITE_PROXY`, api 的代理我看是直接手动一个个配置上去的。
<img width="613" alt="image" src="https://github.com/vbenjs/vue-vben-admin/assets/20253809/eb79a264-f1a7-4c04-b746-17a5630ebe00">

**Comment by fuckphptxt** [comment]
貌似只针对开发环境有效，生产环境配置PROXY代理接口会直接报错404，这也是我一个疑问点，生产环境使用PROXY代理难道无效？

**Reply by GreatAuk** [reply]
生产环境本来就无效的，这个代理依赖 `node` 环境，你觉得生产上会有这个环境吗
